MultiSource/Benchmarks/Ptrdist/yacr2/vcg.c

```diff
@@ -229,7 +229,7 @@ DFSBelowVCG(_Array_ptr<nodeVCGType> VCG : count(channelNets + 1),
 void
 SCCofVCG(_Array_ptr<nodeVCGType> VCG : count(channelNets + 1),
 	     _Array_ptr<ulong> SCC : count(channelNets + 1),
-		 _Array_ptr<ulong> perSCC : count(totalSCC + 1))
```

Comment:
Could you rename this variable from paramSCC to tmpPerSCC, so the connection is clearer?

Reply:
Thanks David. I am still testing these changes. Olden is passing now but Ptrdist seems to be still failing (there is a miscompare in the expected output). I will clean up the variable names once Ptrdist passes.